integration_tests/e2e/goal/addNote.cy.ts

```diff
@@ -8,7 +8,52 @@ context('Add a note', () => {
     cy.task('reset')
     cy.task('stubSignInAsUserWithEditAuthority')
     cy.task('stubAuthUser')
-    cy.task('getPrisonerById')
+    cy.task('getPrisonerById', 'G6115VJ')
```

Comment:
We need to mock 2 calls to getPrisonerByID

The previous mock was for G6115VJ (its the default in the wiremock stub)
But we now need to mock calls for 2 different prisoners. The prisoner number can be passed as an argument like this, which overrides the default